src/components/books.js

```diff
@@ -1,9 +1,46 @@
 import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import AddBooks from './addBooks';
+import { removeBook } from '../redux/books/books';
 
-const Books = () => (
-  <div>
-    Not yet implemented
-  </div>
-);
+const Books = () => {
+  const books = useSelector((state) => state);
+  const dispatch = useDispatch();
+  const displayBooks = [];
+
+  const remove = (e) => {
+    dispatch(removeBook(e.target.id));
+  };
+
+  books.books.forEach((book) => {
+    displayBooks.push(
+      <li key={book.id}>
+        <p>
+          Title:
+          { book.title }
+        </p>
+        <p>
+          Author:
+          { book.author }
+        </p>
+        <button type="button" id={books.books.indexOf(book)} onClick={(e) => { remove(e); }}>Remove Book</button>
+      </li>,
```

Comment:
- [ ] Good job implementing a way to have a book, now let's restructure a little bit, why not use a `<book />` component and call it here, please inside this component add a remove function too! 🐱